torch_frame/data/multi_nested_tensor.py

```diff
@@ -187,6 +190,25 @@ def _apply(self, fn: Callable[[Tensor], Tensor]) -> 'MultiNestedTensor':
 
         return out
 
+    def dim(self) -> int:
+        return 2
+
+    def size(self, dim) -> int:
+        if dim == 0:
+            return self.num_rows
+        elif dim == 1:
+            return self.num_cols
+        else:
+            raise ValueError(f"Dim {dim} does not exist.")
```

Comment:
nit: to be consistent with `torch.Tensor.size()`
```suggestion
            raise IndexError(f"Dimension out of range (expected to be in range of [0, 1], but got {dim}")
```